bitrise.yml

```diff
@@ -1,64 +1,80 @@
 ---
-format_version: "11"
+format_version: '11'
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: flutter
 
-workflows:
-
+pipelines:
   primary:
-    description: |
-      Builds project and runs tests.
+    stages:
+    - analyze-and-test-stage: { }
+    - export-stage: { }
+stages:
+  analyze-and-test-stage:
+    abort_on_fail: true
+    workflows:
+    - analyze: { }
+    - test: { }
+  export-stage:
+    abort_on_fail: true
+    workflows:
+    - export-ios-app: { }
+    - export-android-apk: { }
 
-      Next steps:
-      - Check out [Getting started with Flutter apps](https://devcenter.bitrise.io/en/getting-started/getting-started-with-flutter-apps.html).
+workflows:
+  _setup:
     steps:
-    - git-clone@6: { }
-    - flutter-installer@0: { }
-    - cache-pull@2: { }
-    - flutter-test@1:
+    - git-clone: { }
```

Comment:
We should pin at least the major version of steps

Reply:
Will take care of this in a follow up PR.

Reply:
https://github.com/bitrise-io/getting-started-flutter-sample-app/pull/12/files